lte/gateway/python/integ_tests/s1aptests/test_attach_detach_two_pdns_with_tcptraffic.py

```diff
@@ -30,7 +30,7 @@ def tearDown(self):
         """Cleanup"""
         self._s1ap_wrapper.cleanup()
 
-    def test_attach_detach_two_pdns_with_tcp_traffic(self):
+    def test_attach_detach_two_pdns_with_tcptraffic(self):
```

Comment:
Rename as test_attach_detach_two_pdns_with_tcp_traffic

Reply:
The idea was to align the classname and function name with the filename here for clear debugging in CI with matching string names to search across. This suggested modification will need filename modification and updating of same in defs.mk file. This will be a bigger activity considering many other test cases with similar behavior and can be considered as a lower priority activity for future.